cli/command/container/create_test.go

```diff
@@ -156,7 +156,7 @@ func TestNewCreateCommandWithContentTrustErrors(t *testing.T) {
 			) (container.ContainerCreateCreatedBody, error) {
 				return container.ContainerCreateCreatedBody{}, fmt.Errorf("shouldn't try to pull image")
 			},
-		}, test.EnableContentTrust)
+		}).EnableContentTrust()
```

Comment:
Why this change ? I tend to prefer functionnal argument that "chains" of call… If it's just to not have `test.`, we can *dot import*…

Reply:
I believe @thaJeztah suggested it, I personally don't have any strong opinion on this.

Reply:
By the way this commit actually belongs to the PR before this one e.g. https://github.com/docker/cli/pull/1031